Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -425,35 +424,34 @@ bool ctkDICOMDatabasePrivate::insertConnectionName(const int& dbPatientID,
 }
 
 //------------------------------------------------------------------------------
-bool ctkDICOMDatabasePrivate::updateConnections(const QString& dbPatientID,
-                                                const QStringList& allowList,
-                                                const QStringList& denyList)
+int ctkDICOMDatabasePrivate::updateConnections(const QString& dbPatientID,
+                                               const QStringList& allowList,
+                                               const QStringList& denyList)
 {
   QString connectionsData = this->convertConnectionInfoToJson(allowList, denyList);
-
   QSqlQuery updateConnectionsStatement(this->Database);
   updateConnectionsStatement.prepare("UPDATE Patients SET Connections = :connectionsData WHERE UID = :uid");
   updateConnectionsStatement.bindValue(":connectionsData", connectionsData);
   updateConnectionsStatement.bindValue(":uid", dbPatientID);
 
   if (!loggedExec(updateConnectionsStatement))
   {
-    return false;
+    return -1;
```

Comment:
This method returns -1 or true now. It is probably better if it returns a bool (success) instead.